### DISCUSSION TITLE
Import Tandoor made zip file

### DISCUSSION THREAD

**Discussion by Smolders2007** [TandoorRecipes/recipes#2669] [General]
My computer crashed so I have start from scratch and only have an old (December 2022) saved zip file made with the Tandoor recipes.

I started a new proxmox server and installed Tandoor recipes in docker with portainer. When I try to import this fis zip file in the app section it failed because of a missing resource. The zip file has more than a hundred recipes with its images.

What am I doing wrong?

Regards,
Jean Smolders
[jean.smolders@outlook.com](mailto:jean.smolders@outlook.com)

**Comment by smilerz** [comment]
You can only import from the same version that the export came from.  Unfortunately, I don't believe there is any way to tell the version from the export file.

**Reply by Smolders2007** [reply]
I tried with a new instance and exported a recipe and when trying to import that same recipe I get the exact same error: Error creating a resource, but no log entry giving a clue what resource is missing!

**Reply by smilerz** [reply]
> I tried with a new instance and exported a recipe and when trying to import that same recipe I get the exact same error: Error creating a resource, but no log entry giving a clue what resource is missing!

Did you delete the original recipe first?  What do the logs show?

**Reply by maku1958** [reply]
Hi 
Is there any development in this topic? 
I exported all my 36 Recipes as a zip file.  In order to import them when my server crashes . Eventuelly it did at the start of May. Now after setting up all again I even managed to install tandoor recipes again, so I wanted to import my exported zip file. 
Tandoor recipes explains that the files are no zip files. Is Tandoor not capable of exporting true zip files?
Even the unpacked zip files are not recognized. 
Can I do something about it?

**Reply by maku1958** [reply]
Hi 
I unpacked the the exported "export_date.zip" and found 38 zip files named like 1.zip and so on. 
I had the idea to compress the 1.zip to recipe_1.zip and tried the import. It worked. After I had recompressed all files I also tried to import several zip files at once. There seem to be two limits: 1. only five zip files in one go and 2. the zip files have to be countable , that means in the group of five there have to be files recipe_8.zip p to recipe_12.zip. It wont work if there is no recipe_10.zip.
I hope you understand my explanation. If not feel free to ask.
